### PR TITLE
Updated translation part in project readme

### PR DESCRIPTION
**Pull request by JuanGuerrero09**
### Feature Preview

---
Updated the location mentioned the project readme.

#### PR Checklist

- [x] My code adheres to [AppFlowy's Conventions](https://docs.appflowy.io/docs/documentation/software-contributions/conventions)
- [x] I've listed at least one issue that this PR fixes in the description above.
- [x] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [x] All existing tests are passing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ Proudly wear your T-shirt and show it to us by tagging [@appflowy](https://twitt
 
 [![translation badge](https://inlang.com/badge?url=github.com/AppFlowy-IO/AppFlowy)](https://inlang.com/editor/github.com/AppFlowy-IO/AppFlowy?ref=badge)
 
-To add translations, you can manually edit the JSON translation files in `/frontend/appflowy_flutter/assets/translations`, use the [inlang online editor](https://inlang.com/editor/github.com/AppFlowy-IO/AppFlowy), or run `npx inlang machine translate` to add missing translations.
+To add translations, you can manually edit the JSON translation files in `//frontend/resources/translations`, use the [inlang online editor](https://inlang.com/editor/github.com/AppFlowy-IO/AppFlowy), or run `npx inlang machine translate` to add missing translations.
```
Comment (Xazin) [inline README.md:76]: Seems there is a slash too many

**Review by Xazin — CHANGES_REQUESTED** [review]

**Comment by JuanGuerrero09** [comment]
You are completely right, I've changed it

File: README.md (hunk 1 above)
Comment (JuanGuerrero09) [reply README.md]: If I push the changes on my fork I need to do anything else or just wait for the review?

**Comment by Xazin** [comment]
Hey @JuanGuerrero09, please sign the CLA :)

**Review by Xazin — APPROVED** [review]

**Comment by JuanGuerrero09** [comment]
> Hey @JuanGuerrero09, please sign the CLA :)

Done!

**Pull request merged**